spec.bs

```diff
@@ -3065,17 +3058,21 @@ A <dfn>server auction response</dfn> is a [=struct=] with the following [=struct
   :: A [=list=] of interest group owner and interest group name pairs that bid
      in this auction.
   : <dfn>score</dfn>
-  :: {{double}} or Null. TODO
+  :: {{double}} or Null. For component auctions, contains the desirability

```

Comment:
what's its value for non-component auctions? Same comment for a few other fields below.